server/lib/models/channel.go

```diff
@@ -43,6 +56,18 @@ func NewChannel(dbif Dbif, a *Account, req *proto.ChannelCreateRequest) (*Channe
 	return ch, nil
 }
 
+func FindChannel(dbif Dbif, id proto.UUID) (*Channel, error) {
+	c := &Channel{
+		proto.Model_Channel {
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: proto
* [gotype] too many values in struct literal